### ISSUE TITLE
Torrent hashing fails for some torrents

### ISSUE THREAD

**Issue by buzz** [rakshasa/rtorrent#509]
I have a lot of torrents in rtorrent (~400).
Most of the torrents are fine. But some are not hashing correctly. The files are present and valid (other torrent clients can successfully hash them).
The strange thing: if I trigger a rehash after re-importing a torrent (`^S`) it displays `Hashing: Checking hash [21%]`, counting up but randomly stopping at some point. Smaller torrents are hashed correctly. I have a file that is 1.6 GB that doesn't hash but most are somewhat bigger (several GB). Other files as big as 8 GB have been hashed without problems.

The debug logs don't output anything useful at all…
Anyone?

Arch Linux
0.9.6/0.13.6

**Comment by pyroscope** [comment]
Try looking at it with sysdig, while anything else is stopped (i.e. just have the hashing active).

**Comment by buzz** [comment]
Thanks for the reply.
This actually happened when I put a couple of hundred torrents into the watch directory.

I got around the problem by:
* deleting all torrents
* slowly re-importing torrents through rutorrent front-end in groups of ~30

Currently I'm unable to reproduce the problem as I'm quite happy that stuff works again… :sweat_smile:

PS: Just for the record, I actually noticed that none of the torrents were going into Seeding state. They were all stuck either at Incomplete because hashing didn't work or Complete.
